frontend/src/reducers/varianceReducer.js

```diff
@@ -12,11 +13,17 @@ const varianceReducer = (state = initialState, action) => {
         ...state,
         mineVariances: action.payload.records,
       };
+    case actionTypes.STORE_VARIANCE:
+      return {
+        ...state,
+        variance: action.payload,
```

Comment:
Just curious if this the way to do this? I'm guessing this is the super strict redux pattern? but wouldn't it make more sense to add it to some collection then have a selector that is able to pull out a specific variance? Cause if a separate component called 'getVarianceById` it will overwrite this value? seems like a lots of work to set up a place in the store for a single item (likely from a set), and we might want to load multiple specific ones at some point. 

Tailings Expected documents has a similar workflow, but it reloads from the whole mine.

Reply:
There is a `mineVariance` array stored in the state as well, which houses all the variances attached to a mine.

Previously I was able to access the individual variances in the array without needing `fetchVarianceById` by mapping through the array and passing in the entire variance into the modal when it was opened.
That works great until you want to update the variance inside the modal without closing the modal.

Does that make sense? I can't think of a situation where the user would be updating the store in a different part of the application that would interfere with this functionality.